### PR TITLE
MS Entra Enhancement

### PR DESCRIPTION
**Pull request by eepstain**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-dc.paloaltonetworks.com/browse/CIAC-11777)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MicrosoftEntraID/README.md [hunk 1]
```diff
@@ -51,6 +55,19 @@ To connect Cortex XSIAM to the Azure Event Hub, follow the below steps.
 
 More information can be found [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Administrator-Guide/Ingest-Logs-from-Microsoft-Azure-Event-Hub?tocId=yjPDSlvRYtlNncGBLHOzvw).
 
+#### Native O365 Collector
+
+To collect logs for the **msft_azure_ad_raw** and **msft_azure_ad_audit_raw** datasets, you will need to create or configure the Office 365 collector.
+Use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Pro-Administrator-Guide/Ingest-Logs-from-Microsoft-Office-365).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:61]:
```suggestion
Use the information described [here](https://docs-cortex.paloaltonetworks.com/r/Cortex-XDR/Cortex-XDR-Documentation/Ingest-logs-from-Microsoft-Office-365).
```

File: Packs/MicrosoftEntraID/README.md [hunk 2]
```diff
@@ -51,6 +55,19 @@ To connect Cortex XSIAM to the Azure Event Hub, follow the below steps.
...
+
+To access the Office 365 Native Collector on your Cortex XSIAM tenant:
+1.  On the left panel, click **Settings** &rarr; **Data Sources**
+2.  At the top-right corner, click **Add Data Source**
+3.  Search for **Office 365** and click **Connect**.
+4.  Input the mandatory and required credentials for **Azure AD** and **Azure AD Audit**:
+    * Mark the checkboxes **Azure AD Authentication Logs** &rarr; **Collect all sign-in event types**.
+    * Mark the checkboxes **Azure AD Audit Logs**.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftEntraID/README.md:69]:
```suggestion
    * Mark the **Azure AD Authentication Logs** &rarr; **Collect all sign-in event types** checkbox.
    * Mark the **Azure AD Audit Logs** checkbox.
```

File: Packs/Office365/README.md [hunk 3]
```diff
@@ -11,10 +11,13 @@ To access the Office 365 Native Collector on your Cortex XSIAM tenant:
 2.  At the top-right corner, click **Add Data Source**
 3.  Search for **Office 365** and click **Connect**.
 
+**Pay Attention**:
+In order to normalize **Azure AD** (msft_azure_ad_raw) and **Azure AD Audit** (msft_azure_ad_audit_raw) logs, please install the Microsoft Entra ID pack.
```
Comment (ShirleyDenkberg) [inline Packs/Office365/README.md:15]:
```suggestion
In order to normalize **Azure AD** (msft_azure_ad_raw) and **Azure AD Audit** (msft_azure_ad_audit_raw) logs, install the Microsoft Entra ID pack.
```

**Comment by ShirleyDenkberg** [comment]
@eepstain Doc review completed.

**Review by OBavly — APPROVED** [review]

**Pull request merged**